AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -506,7 +506,18 @@ typedef CALayer *(^ASDisplayNodeLayerBlock)();
 @interface ASDisplayNode (UIViewBridge)
 
 - (void)setNeedsDisplay;    // Marks the view as needing display. Convenience for use whether view is created or not, or from a background thread.
-- (void)setNeedsLayout;     // Marks the view as needing layout.  Convenience for use whether view is created or not, or from a background thread.
+
+/**
+ * Marks the view as needing layout. Convenience for use whether view is created or not, or from a background thread.
```

Comment:
Let's rephrase this to be "Marks the node as needing layout.  Convenience for use whether the view / layer is loaded or not.  Safe to call from a background thread."

The original comment likely dates to before layer-backing support.  Now with the expanded abstractions related to layout in AsyncDisplayKit, we can start referring to "node" layout as a more distinct concept than just normal view / layer layout.